### ISSUE TITLE
`LabelText` is inline, but really should be a `block` 🤔

### ISSUE THREAD

**Issue by TejasQ** [contiamo/operational-ui#727]
I'm really not sure why we use a `span` for `LabelText`. It is inline, but all of our designs and implementations have labels on separate lines above other elements. Consider,

# Status Quo
![image](https://user-images.githubusercontent.com/9947422/45148204-0547cd00-b1c7-11e8-8fe8-e1d18211a460.png)

# Expected
![image](https://user-images.githubusercontent.com/9947422/45148331-52c43a00-b1c7-11e8-8b38-ceae5b53a5b4.png)

This issue humbly requests fixing this by making `LabelText` a block-level element.

**Comment by adeelibr** [comment]
Can I work on this?

**Comment by peterszerzo** [comment]
I think it makes sense to take care of both this issue and #726 by both converting `LabelText` from https://github.com/contiamo/operational-ui/blob/master/src/utils/mixins.ts#L37 to a block-style element, as well as moving it into its own folder `src/LabelText` and making it a public export.

**Comment by adeelibr** [comment]
Can there be 2 PR for this? One for the label to block conversion for `LableText` and the other where we move it into it's seperate folder `src/LabelText` or should both of them be done in the same PR?

**Comment by peterszerzo** [comment]
@adeelibr that's completely up to you. It is a teeny bit safer to split up into two PR's, but for a change this small, I'm not sure it's worth the bureaucracy :)

(also, thanks for giving us a hand on this!)

**Comment by adeelibr** [comment]
@peterszerzo I don't mind doing it in 2 PR's 😅 Once this PR is done. I'd be more then happy to create a new pull request for `LabelText` to move into it's own folder.

**Comment by adeelibr** [comment]
@peterszerzo The first PR was merged, can I now make another PR to move it into a separate module. If no one is working on it?

**Comment by adeelibr** [comment]
The first PR was done https://github.com/contiamo/operational-ui/pull/753  where I converted the element to `block` level. I am still waiting for go ahead to make the next PR to move this to a separate file. @TejasQ

**Comment by TejasQ** [comment]
Ah, you're waiting on @peterszerzo. Cool!

**Comment by adeelibr** [comment]
yup :sw

**Comment by peterszerzo** [comment]
@adeelibr you're welcome to go ahead with the other branch.
